BusinessTampereTrafficMonitoring/traffic_lights/api_client.py

```diff
@@ -69,7 +81,12 @@ def update_device_state(self, device: str):
         return events
 
     def store(self, events: List):
-        """Stores events into the database."""
+        """
+        Stores events into the database.
+        #Parameter:
+        # argument1: events (List[])
+
```

Comment:
Try to be consistent with the empty lines. Most docstrings end immediately after the text, others (like this one) have an extra empty line before the closing `"""`. Personally I think the format without the newline is better, but most important thing is that they all consistently follow the same format.